concent-builder/template/jenkins/jenkins.model.JenkinsLocationConfiguration.xml.j2

```diff
@@ -0,0 +1,5 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<jenkins.model.JenkinsLocationConfiguration>
+  <adminAddress>{{ jenkins_admin_email_address }}</adminAddress>
+  <jenkinsUrl>http://{{ jenkins_url_address }}/</jenkinsUrl>
```

Comment:
`jenkins_url_address` -> `jenkins_url`. And I'd include the `http://` part in the variable too.